src/wp-includes/rest-api/class-wp-rest-server.php

```diff
@@ -1399,6 +1399,11 @@ public function get_data_for_route( $route, $callbacks, $context = 'view' ) {
 				$endpoint_data['args'] = array();
 
 				foreach ( $callback['args'] as $key => $opts ) {
+					if ( is_string( $opts ) ) {
```

Comment:
Do we need this check here? It looks like the value will just end up getting stripped out by the `array_intersect_key` check, right?